schema/schema.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE IF NOT EXISTS Calendar (
+	id INT AUTO_INCREMENT PRIMARY KEY,
+	user_fk INT NULL,
```

Comment:
I don't think the user_fk field is necessary anymore, since the User table has a primary_calendar_fk field

Reply:
ok, I will drop.